app/components/branch-row.js

```diff
@@ -6,15 +6,15 @@ import config from 'travis/config/environment';
 export default Ember.Component.extend({
   routing: Ember.inject.service('-routing'),
   tagName: 'li',
-  classNameBindings: ['build.last_build.state'],
+  classNameBindings: ['branch.last_build.state'],
```

Comment:
I’m generally trepidatious regarding giant renames without proper tests to ensure functionality has been preserved! But you do seem to have covered all the uses.